fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PruneHDFSScanColumnRule.java

```diff
@@ -134,21 +134,17 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
     }
 
     private boolean containsMaterializedColumn(LogicalScanOperator scanOperator, Set<ColumnRefOperator> scanColumns) {
-        if (scanOperator instanceof LogicalHiveScanOperator) {
```

Comment:
how about adding a unit test case for hudi?

Reply:
done